lesson03/util/util.go

```diff
@@ -0,0 +1,58 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+)
+
+func FindAverage(arr []int) float64 {
+	if len(arr) == 0 {
+		fmt.Println("empty array")
+		return 1
+	}
+	sum := 0.0
+	for _, elem := range arr {
+		sum += float64(elem)
+	}
+	return sum / float64(len(arr))
+}
+
+func Max(sarr []string) string {
+	if len(sarr) == 0 {
+		return "empty array"
+	}
+	maxstr := 0
+	for ind, val := range sarr {
+		if len(sarr[maxstr]) < len(val) {
+			maxstr = ind
+		}
+	}
+	return sarr[maxstr]
+}
+
+func Reverse(arr []int64) []int64 {
+	l := len(arr)
+	if l == 0 {
+		fmt.Println("empty array")
+		return 1
```

Comment:
This is a wrong return type, which will result in compiler error, it should return a slice, not an integer. 

Same as with other functions above, this check is unnecessary, the reverse of empty slice is an empty slice.